creating-joy/src/main/java/com/technocrats/creatingjoy/dto/UserDTO.java

```diff
@@ -1,4 +1,116 @@
 package com.technocrats.creatingjoy.dto;
 
+        import lombok.AllArgsConstructor;
+
+        import lombok.NoArgsConstructor;
+
+        import org.springframework.stereotype.Component;
+
+
+
+
+@NoArgsConstructor
+@AllArgsConstructor
 public class UserDTO {
-}
+
+
+    private String firstName;
+    private String lastName;
+    private String userName;
+    private String phoneNumber;
+    private String password;
+    private String houseNumber;
+    private String street;
+    private String city;
+    private String state;
+
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public String getPhoneNumber() {
```

Comment:
Use @Getter and @Setter just like you did for constructors

Reply:
Done.